### DISCUSSION TITLE
How enable dark-mode for all page?

### DISCUSSION THREAD

**Discussion by sheldygg** [aminalaee/sqladmin#844] [Q&A]
<img width="1565" alt="image" src="https://github.com/user-attachments/assets/066f708c-d8b0-4ea8-8fa4-e6297540ddad">
I have add `dark-bs-theme` attr to html, but list still is white, why?

**Comment by sheldygg** [comment]
Okay, we must update version of tabler.css
sqladmin use version `1.0.0-beta17`, latest is `1.0.0-beta21`
After [updating version](https://cdn.jsdelivr.net/npm/@tabler/core@latest/dist/css/tabler.min.css) I still see some white things
<img width="327" alt="image" src="https://github.com/user-attachments/assets/c7143dd8-a5ab-4b9a-9554-d6d05a766328">

And then sidebar in white version is not green or blue (idk what is this color :))
<img width="251" alt="image" src="https://github.com/user-attachments/assets/cbedbc8f-9778-4b9f-b12f-299f5f5bd29b">

What we must do with this?

**Comment by sheldygg** [comment]
We can remove `btn-light` from list.html template, color will be applied dynamically.
I also found a white element when choosing a date.

**Comment by sheldygg** [comment]
I played around with sqladmin templates a bit and came up with this design (light and dark theme support)

<img width="1784" alt="image" src="https://github.com/user-attachments/assets/d373809a-54e3-40a9-b862-cdea287a58b3">

I think this much pretty then current sqladmin design, @aminalaee what you think about that?

**Comment by mmzeynalli** [comment]
Seems nice. Could you open PR for this?
